src/search/FindInFiles.js

```diff
@@ -93,7 +93,19 @@ define(function (require, exports, module) {
     }
 
     function nodeFileCacheComplete(event, numFiles, cacheSize) {
-        var projectName = ProjectManager.getProjectRoot().name || "noName00";
+        if (/\/test\/SpecRunner\.html$/.test(window.location.pathname)) {
```

Comment:
We cannot be certain what the git clone is named on the system. Also, it's very likely that a Brackets Shell window that has a SpecRunner.html loaded, which in turn loaded the rest of Brackets, actually belongs to Brackets ;)